pkg/controller/node/node_controller.go

```diff
@@ -376,7 +376,13 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 	}
 
 	if machineconfigpool.Status.Configuration.Name == "" {
-		return fmt.Errorf("Empty Current MachineConfig")
+		delay := 5 * time.Second
+		// Previously we spammed the logs about empty pools.
+		// Let's just pause for a bit here to let the renderer
+		// initialize them.
+		glog.Infof("Pool %s is unconfigured, pausing %v for renderer to initialize", delay)
+		time.Sleep(delay)
```

Comment:
```
# github.com/openshift/machine-config-operator/pkg/controller/node
pkg/controller/node/node_controller.go:383: Infof format %v reads arg #2, but call has 1 arg
pkg/controller/node/status.go:32: Infof format %s has arg newPool.Status of wrong type github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1.MachineConfigPoolStatus
```

Reply:
OK I dropped the status patch, did that at the last second and hadn't tested it.  Doing that "right" is probably changing the operator to do the logging, as it already has the code to render things to a string.